components/Cards/index.js

```diff
@@ -17,3 +17,43 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+axios
+  .get("https://lambda-times-backend.herokuapp.com/articles")
+  .then(response => {
+    Object.values(response.data.articles).forEach(item => {
+      item.forEach(items => {
+        getarticles.append(createArticles(items));
+      });
+    });
+  })
+  .catch(error => {
+    console.log(error);
+  });
+
+function createArticles(article) {
+  const container = document.createElement("div"),
+    headLine = document.createElement("div"),
+    author = document.createElement("div"),
+    img = document.createElement("div"),
+    imgA = document.createElement("img"),
+    spans = document.createElement("span");
```

Comment:
Is there any reason you declared them like this with only one `const` rather then a `const` for each one?